_config.yml

```diff
@@ -23,12 +23,7 @@ description: >- # this means to ignore newlines until "baseurl:"
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://www.rubynepal.org" # the base hostname & protocol for your site, e.g. http://example.com
 permalink: /blog/:title
-twitter_username: "ruby_nepal"
-github_username: "rubynepal"
-facebook_username: "RubyNepal.org"
-slack_username: "rubynepal"
-meetup_username: "Nepal-Ruby-Users-Group"
-youtube_channelname: "UCoRCDWyFmaby7F4qWRUHgAQ"
+twitter_username: "ruby_nepal" # used in post sharing
```

Comment:
If I find twitter link here, I would expect to find other social links here as well?